### PR TITLE
fix deploy-logging to always use CLO make deployment

### PR DESCRIPTION
**Pull request by jcantrill**
This PR fixes the smoke test:

* Updates deploy-logging.sh to rely on CLO make deploy target

### CODE REVIEW

File: hack/deploy-logging.sh [hunk 1]
```diff
@@ -331,14 +329,10 @@ deploy_logging_using_olm() {
...
-    # make deploy-no-build
-    EO_IMAGE=${EO_IMAGE:-$( construct_image_name elasticsearch-operator latest )}
-    CLO_IMAGE=${CLO_IMAGE:-$( construct_image_name cluster-logging-operator latest )}
     pushd $CLO_DIR > /dev/null
     cp manifests/${MASTER_VERSION}/cluster-logging.*.clusterserviceversion.yaml manifests/${MASTER_VERSION}/cluster-logging.*.clusterserviceversion.yaml.orig
-    update_images_in_clo_yaml manifests/${MASTER_VERSION}/cluster-logging.*.clusterserviceversion.yaml $CLO_IMAGE
-    REMOTE_CLUSTER=true REMOTE_REGISTRY=true NAMESPACE=$LOGGING_NS \
-        IMAGE_OVERRIDE="$CLO_IMAGE" EO_IMAGE_OVERRIDE="$EO_IMAGE" make deploy-no-build
+    update_images_in_clo_yaml manifests/${MASTER_VERSION}/cluster-logging.*.clusterserviceversion.yaml 
+    make deploy
```
Comment (vimalk78) [inline hack/deploy-logging.sh:335]: we can capture the `$?` value and throw error in case we ever remove/change `make deploy` target in CLO Makefile

Comment (jcantrill) [reply hack/deploy-logging.sh:335]: wont we know this by the fact the operation fails?

File: hack/deploy-logging.sh [hunk 2]
```diff
@@ -420,26 +410,26 @@ else
 fi
 
 if [ "${LOGGING_DEPLOY_MODE:-install}" = install ] ; then
-    CLO_DIR=${CLO_DIR:-$GOPATH/src/github.com/openshift/cluster-logging-operator}
-    if [ ! -d $CLO_DIR ] ; then
-        CLO_DIR=$ARTIFACT_DIR/clo
-        get_operator_files $CLO_DIR cluster-logging-operator ${CLO_REPO:-openshift} ${CLO_BRANCH:-master}
+    CLO_DIR=$GOPATH/src/github.com/openshift/cluster-logging-operator
+    EO_DIR=$GOPATH/src/github.com/openshift/elasticsearch-operator
+    if [ ! -d $GOPATH/src/github.com/openshift ] ; then
```
Comment (blockloop) [inline hack/deploy-logging.sh:415]: unnecessary `if` check. `mkdir` will noop if the dir already exists

File: openshift/ci-operator/build-image/cr.yaml [hunk 3]
```diff
@@ -14,10 +14,10 @@ spec:
       redundancyPolicy: "ZeroRedundancy"
       resources:
         limits:
-          memory: 3Gi
+          memory: 2Gi
         requests:
-          cpu: 500m
-          memory: 3Gi
+          cpu: 100m
+          memory: 2Gi
```
Comment (blockloop) [inline openshift/ci-operator/build-image/cr.yaml:20]: Can you comment on the reason for these changes? Are they arbitrary?

File: hack/deploy-logging.sh (hunk 2 above)
Comment (vimalk78) [reply hack/deploy-logging.sh:415]: mkdir will fail if dir already exists. `$?` is non zero
the script sets `set -euo pipefail` which makes script to exit if any operation fails.

Comment (blockloop) [reply hack/deploy-logging.sh:415]: Not when you use the `-p` flag

```
$ mkdir -p $PWD
$ echo $?
0
```

File: openshift/ci-operator/build-image/cr.yaml (hunk 3 above)
Comment (jcantrill) [reply openshift/ci-operator/build-image/cr.yaml:20]: We have no specific needs for these values.  I lowered them to ensure we could land the ES nodes without having to bump the OCP sizes.

**Comment by blockloop** [comment]
/lgtm
my nits can be addressed later

**Comment by jcantrill** [comment]
overriding bz label as this is a change to address broken tests

**Pull request merged**